Docs/Migration-INI.md

```diff
@@ -0,0 +1,972 @@
+```
+
+2. **OPTIONAL (YR+Phobos)** Add sections:
+
+```ini
+[GameOptionsPanelExtraControls]
+; Only available with Phobos
+0=chkTooltipsExtra:SettingCheckBox
+1=chkPrioritySelection:SettingCheckBox
+2=chkBuildingPlacement:SettingCheckBox
+
+[chkTooltipsExtra]
+Location=24,151, ;12,151
+Text=Sidebar Tooltip Descriptions
+ToolTip=Enables additional information in sidebar tooltips.
+DefaultValue=true
+ParentCheckBoxName=chkTooltips
+ParentCheckBoxRequiredValue=true
+SettingSection=Phobos
+SettingKey=ToolTipDescriptions
+
+[chkPrioritySelection]
+Location=242,54
+Text=Mass Selection Filtering
+ToolTip=If enabled, non-combat units are not selected if mass-selecting together with combat units.
+DefaultValue=false
+SettingSection=Phobos
+SettingKey=PrioritySelectionFiltering
+
+[chkBuildingPlacement]
+Location=242,78
+Text=Show Building Placement Preview
+ToolTip=If enabled, shows a preview image of the building when placing it.
+DefaultValue=false
+SettingSection=Phobos
+SettingKey=ShowBuildingPlacementPreview
+```
+
+3. Add sections:
+
+```ini
+[lblPlayerName]
+Location=12,195
+
+[tbPlayerName]
+Location=113,193
+
+[lblNotice]
+Location=12,220
+
+[btnConfigureHotkeys]
+Location=12,290
+
+[chkDisablePrivateMessagePopup]
+Location=12,138
+Text=Disable private message pop-ups
+
+[chkAllowGameInvitesFromFriendsOnly]
+Location=276,68
+Text=Only receive game invitations@from friends
+
+[lblAllPrivateMessagesFrom]
+Location=276,138
+
+[ddAllowPrivateMessagesFrom]
+Location=470,137
+
+[gameListPanel]
+Location=0,200
+```
+
+## Create new `PlayerExtraOptionsPanel.ini`
+
+A new panel that allows for convenient match setup has been added in the
+game lobby.
+
+```ini
+[btnClose]
+Location=220,0
+Size=18,18
+
+[lblHeader]
+Location=12,6
+
+[chkBoxForceRandomSides]
+Location=12,28
+
+[chkBoxForceRandomColors]
+Location=12,50
+
+[chkBoxForceRandomTeams]
+Location=12,72
+
+[chkBoxForceRandomStarts]
+Location=12,94
+
+[chkBoxUseTeamStartMappings]
+Location=12,130
+
+[btnHelp]
+Location=160,130
+
+[lblPreset]
+Location=12,156
+
+[ddTeamStartMappingPreset]
+Size=157,21
+Location=65,154
+
+[teamStartMappingsPanel]
+Location=12,189
+```
+
+## Appendix
+
+For completion's sake, below are additional steps required for a complete
+migration (beyond INI changes) to client version [2.11.0.0][client].
+
+### Replace binary files
+
+1. Replace contents of `Resources/Binaries` and `Resources/Compatibility` with
+   new files. This directory contains the .NET Framework 4.8 version
+   of the client.
+2. Copy contents of downloaded `BinariesNET8` into a new directory
+   `Resources/BinariesNET8`. This directory contains the .NET 8 version
+   of the client.
+3. Replace your launcher executable in game directory (whetever it's called,
+   `YRLauncher.exe`, `MentalOmegaLauncher.exe`, etc.) with version
+   [2.0.7](https://github.com/CnCNet/dta-mg-client-launcher/releases/tag/v2.0.7).
+4. Replace the second-stage updater in `Resource/Updater` with version

```

Comment:
I think we can stress that both second-stage updaters (.NET  4.8 and .NET 8.0) can be copied in `Resource/Updater`. 

The following tree view should be helpful to readers.

```txt
<game dir>/Resources/Updater
├── Rampastring.Tools.dll # common dependencies of .NET 4.8/.NET 8.0 second-stage updater
├── SecondStageUpdater.dll # .NET 8.0 second-stage updater
├── SecondStageUpdater.exe # .NET 4.8 second-stage updater
├── SecondStageUpdater.exe.config  # .NET 4.8 second-stage updater
└── SecondStageUpdater.runtimeconfig.json # .NET 8.0 second-stage updater
```